### PR TITLE
added validation for requestDto

### PR DESCRIPTION
**Pull request by yanna-stepanova**

### CODE REVIEW

File: src/main/java/com/yanna/stepanova/dto/CreateBookRequestDto.java [hunk 1]
```diff
@@ -1,6 +1,10 @@
 package com.yanna.stepanova.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 
-public record CreateBookRequestDto(String title, String author, BigDecimal price,
+public record CreateBookRequestDto(@NotNull String title,
+                                   @NotNull String author,
+                                   @NotNull @Min(0) BigDecimal price,
```
Comment (Elena-Bruyako) [inline src/main/java/com/yanna/stepanova/dto/CreateBookRequestDto.java]:
```suggestion
                                   @NotNull @Positive BigDecimal price,
```

File: src/main/java/com/yanna/stepanova/dto/CreateBookRequestDto.java [hunk 2]
```diff
@@ -1,6 +1,10 @@
 package com.yanna.stepanova.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 
-public record CreateBookRequestDto(String title, String author, BigDecimal price,
+public record CreateBookRequestDto(@NotNull String title,
```
Comment (Elena-Bruyako) [inline src/main/java/com/yanna/stepanova/dto/CreateBookRequestDto.java]: Use `@NotBlank` for String

Comment (Elena-Bruyako) [inline src/main/java/com/yanna/stepanova/dto/CreateBookRequestDto.java]: Didn't find isbn field

**Review by Elena-Bruyako — CHANGES_REQUESTED** [review]
Good job! Let's improve your solution!

**Review by Rommelua — APPROVED** [review]

**Pull request closed without merging**